test/preparation/test_protonate.py

```diff
@@ -141,27 +159,39 @@ def test_pdb2pqr_protonate_pdb_4NKK():
     fs.safe_rm(actual_pqr)
     assert not os.path.exists(actual_pqr)
 
+
 def test_protonate_ligand_with_pybel():
     """test that protonate_ligand_with_pybel() works without exceptions
     verify the protonation with manual curated atom list  and residue name/idx.
     also test normal behavior of removing int_dir"""
     test_pdb = f"{DATA_DIR}/FAcD-FA-ASP_rmH.pdb"
     int_lig_dir = f"{WORK_DIR}/ligand_test"
+    assert not os.path.isdir(int_lig_dir)
 
     stru = sp.get_structure(test_pdb)
+
     prot.protonate_ligand_with_pybel(stru, 7.0, int_lig_dir)
-    assert stru.ligands[0].atom_name_list == ["C", "F", "O", "CH3", "OXT", "H", "H1"]
+
+    assert stru.ligands[0].atom_name_list == [
+        "C", "F", "O", "CH3", "OXT", "H", "H1"
+    ]
     assert stru.ligands[0].idx == 298
     assert stru.ligands[0].name == "FAH"
+
+    fs.safe_rmdir(int_lig_dir)
```

Comment:
We should not remove it here since part of the test is to check if the function removes the dir itself.

Reply:
Removed. Test still passes.